spec/observables/from-promise-spec.js

```diff
@@ -1,21 +1,115 @@
 /* globals describe, it, expect */
 var Rx = require('../../dist/cjs/Rx');
 var Observable = Rx.Observable;
-var Promise = require('promise');
 
 describe('Observable.fromPromise', function () {
-  it('should emit one value from that promise', function (done) {
+  it('should emit one value from a resolved promise', function (done) {
     var promise = Promise.resolve(42);
     Observable.fromPromise(promise)
       .subscribe(function (x) {
         expect(x).toBe(42);
-      }, null,
+      }, done.fail,
       function (x) {
         expect(x).toBe(undefined);
```

Comment:
Those are likely leftovers from when `complete` actually did pass a value per the es-observable spec.